api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -247,15 +247,25 @@ private void validateStringLength(String field, String fieldName, int max, int m
     }
   }
 
-  private void validateProfileFields(Profile profile) {
+  private void validateAndCleanProfile(Profile profile) throws BadRequestException {
```

Comment:
This is one minor functional change I made along the way... I felt we should probably have more of the profile object validation centralized in this method, which is called during both the createProfile and updateProfile flows.

Reply:
This seems reasonable.